AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -4815,7 +4816,7 @@ deepl.com##div[class^="cookieBanner-"]
 kaspersky.es,kaspersky.de##section[class^="CookieBanner-"]
 hannovermesse.de##.notifier
 rws.com##.rws-cookie-bar
-inf-shop.de,fun-corner.de,jimmygreen.com##.gdprModal
+haineshop.ro,inf-shop.de,fun-corner.de,jimmygreen.com##.gdprModal
```

Comment:
It has to be rewritten due to `haineshop.ro#$#body { overflow: visible !important; }`

Reply:
Fixed https://github.com/AdguardTeam/AdguardFilters/commit/06dd98c2ccaa3cb6b15e4ea5a6f105d83b7dca11